firmware/steering/kia_soul_ps/steering_control_module.ino

```diff
@@ -28,8 +28,9 @@
 #include "PID.h"
```

Comment:
Once again, if you are including globals.h, you may not need all of these include files.